clouddriver-core/src/test/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskSpec.groovy

```diff
@@ -36,7 +36,7 @@ class JedisTaskSpec extends Specification {
 
   void setup() {
     repository = Mock(RedisTaskRepository)
-    task = new JedisTask('666', System.currentTimeMillis(), repository, false)
+    task = new JedisTask('666', System.currentTimeMillis(), repository, "owner", false)
```

Comment:
Maybe add a test proving it works ok with missing owner ids, and one proving an instance just returns its own tasks?